test/it/group-list.js

```diff
@@ -32,6 +32,6 @@ describe('Group API tests', () => {
     expect(groupPresent).to.equal(true);
 
     // 3. Delete the group
-    await client.deleteGroup(createdGroup.id);
+    await utils.cleanup(client, null, createdGroup);
```

Comment:
Thanks for the info. Makes sense if we want to run tests individually, but am curious how are are expecting them to run these tests.

If a developer is looking to debug one of the tests, wouldn't they just update the `npm test` script accordingly? If so, I think separating all of our **after** tests into a post-script, say: 
```json
"posttest":  "node ./it/test/clean.js"
```

^ This does have some drawbacks, but I personally like it.